File: MoM.Module/Dtos/SiteSettingAuthenticationDto.cs
namespace MoM.Module.Dtos
{
    public class SiteSettingAuthenticationDto
    {
        public SiteSettingAuthenticationFacebookDto facebook { get; set; }
        public SiteSettingAuthenticationGoogleDto google { get; set; }
        public SiteSettingAuthenticationMicrosoftDto microsoft { get; set; }
        public SiteSettingAuthenticationTwitterDto twitter { get; set; }
    }
}

File: MoM.Web/app/dtos/SiteSettingAuthenticationDto.ts
import {SiteSettingAuthenticationFacebookDto} from "./SiteSettingAuthenticationFacebookDto";
import {SiteSettingAuthenticationGoogleDto} from "./SiteSettingAuthenticationGoogleDto";
import {SiteSettingAuthenticationMicrosoftDto} from "./SiteSettingAuthenticationMicrosoftDto";
import {SiteSettingAuthenticationTwitterDto} from "./SiteSettingAuthenticationTwitterDto";

export interface SiteSettingAuthenticationDto {
    facebook: SiteSettingAuthenticationFacebookDto;
    google: SiteSettingAuthenticationGoogleDto;
    microsoft: SiteSettingAuthenticationMicrosoftDto;
    twitter: SiteSettingAuthenticationTwitterDto;
}
